### PR TITLE
make it better

### PR DESCRIPTION
**Pull request by zenthiccc**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        type: "light",
+      },
+    })
+  );
+
+  useEffect(() => {
+    const checkLogin = async () => {
+      const token = localStorage.getItem("tokenStore");
+      if (token) {
+        const verified = await axios.get("/users/verify", {
```
Comment (sheensantoscapadngan) [inline client/src/App.js]: ideally, the api call should be inside an "api" folder where I export lang nimo siya as function para function call lang basically ag imo buhaton sa frontend na code

File: client/src/App.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          headers: { Authorization: token },
+        });
+        console.log(verified);
```
Comment (sheensantoscapadngan) [inline client/src/App.js]: Remove console.log statement

File: client/src/components/Login.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
+import React, { useState } from "react";
+import axios from "axios";
+import Typography from "@material-ui/core/Typography";
+import Container from "@material-ui/core/Container";
+import Paper from "@material-ui/core/Paper";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
```
Comment (sheensantoscapadngan) [inline client/src/components/Login.js]: import as one with deconstruction { }

File: client/src/components/Login.js [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+export default function Login({ setIsLogin }) {
+  const [user, setUser] = useState({ name: "", email: "", password: "" });
```
Comment (sheensantoscapadngan) [inline client/src/components/Login.js]: for dictionaries with more than 3 values, always use next line e.g.

values = {
    name: "Sheen",
    age: 15,
    birthday: "March 7, 2003",
}

Comment (sheensantoscapadngan) [reply client/src/components/Login.js]: ^ that should be properly indented

File: client/src/components/Login.js [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+  const onChangeInput = (e) => {
+    const { name, value } = e.target;
+    setUser({ ...user, [name]: value });
+    setError("");
+  };
+
+  const registerSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const res = await axios.post("/users/register", {
```
Comment (sheensantoscapadngan) [inline client/src/components/Login.js]: extract this to an api folder. Maybe something like userApi.js inside a folder called "api"

File: client/src/components/notes/CreateNote.js [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+    try {
+      const token = localStorage.getItem("tokenStore");
+      if (token) {
+        const { title, content } = note;
+        const newNote = {
+          title,
+          content,
+        };
+
+        await axios.post("/api/notes", newNote, {
```
Comment (sheensantoscapadngan) [inline client/src/components/notes/CreateNote.js]: same for the ones above

File: controllers/noteCtrl.js [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+      const { title, content } = req.body;
+      const newNote = new Notes({
+        title,
+        content,
+        user_id: req.user.id,
+        name: req.user.name,
+      });
+      await newNote.save();
+      res.json({ msg: "Created a Note" });
+    } catch (err) {
```
Comment (sheensantoscapadngan) [inline controllers/noteCtrl.js]: console.error(err);

File: client/src/css/index.css [hunk 8]
```diff
@@ -0,0 +1,203 @@
...
+  margin-left: -20px;
+  width: auto;
+}
+.my-masonry-grid_column {
+  padding-left: 20px;
+  background-clip: padding-box;
+}
+.my-masonry-grid_column > div {
+  margin-bottom: 30px;
+}
```
Comment (sheensantoscapadngan) [inline client/src/css/index.css]: separate to individual css stuffs

File: README.md [hunk 9]
```diff
@@ -1 +1,27 @@
-pogi ako 
\ No newline at end of file
+![notism](https://user-images.githubusercontent.com/72091205/122147983-72b2dd00-ce8c-11eb-808d-70865202da34.png)
+
+## Notism
```
Comment (maxellmilay) [inline README.md:3]: test review

Comment (andrytumacole) [reply README.md:3]: test review (1)

**Pull request closed without merging**